.github/workflows/release.yml

```diff
@@ -16,8 +16,8 @@ on:
       - v*
 
 env:
-  GHJK_VERSION: "2725af8"
-  DENO_VERSION: "1.43.6"
+  GHJK_VERSION: "6a9ecda"
+  GHJK_ENV: "ci"
```

Comment:
comment for ghjk, should not ghjk autodetect where the "CI" env is set?

Reply:
Should be possible. Will add that.